cmake/projects/Boost/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -188,8 +188,16 @@ else()
   set(copy_mpi_command "")
 endif()
 
+if(BUILD_SHARED_LIBS) #if using shared libs overide boost config
+  set(Boost_USE_STATIC_LIBS OFF)
```

Comment:
> as the user may have manually set the variable

He should not so I think it's okay to do override. If user will modify such variables there is a risk of finding system libraries instead of libraries from Hunter.

> could set Boost_USE_STATIC_LIBS to ON as default in hunter.cmake in the project boost folder. Would this need to be added to each hunter.cmake in the subfolders?

Actually I think this one should be removed :) Need to be moved to the BoostConfig.cmake.in and to the pkg.boost wiki in case for the backward compabible usage.